yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -62,7 +62,7 @@ describe('AVM simulator: injected bytecode', () => {
     const results = await new AvmSimulator(context).executeBytecode(bytecode);
     expect(results.reverted).toBe(true);
     expect(results.output).toEqual([]);
-    expect(results.revertReason?.name).toEqual('OutOfGasError');
+    expect(results.revertReason?.message).toEqual('Not enough L2GAS gas left');
```

Comment:
`name` is no longer available here since `revertReason` is not an `AvmExecutionError` anymore but instead an `ExecutionError` (or actually an `AvmRevertReason`)